backend/app/services/memory/utils.py

```
@@ -0,0 +1,161 @@
+
+    # Build memory list
+    memory_lines = []
+    for idx, memory in enumerate(memories, start=1):
+        # Extract created_at from metadata if available
+        created_at = memory.metadata.get("created_at", "")
+        if created_at:
+            try:
+                # Format: YYYY-MM-DD
+                dt = datetime.fromisoformat(created_at.replace("Z", "+00:00"))
+                date_str = dt.strftime("%Y-%m-%d")
+            except Exception:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)